python/smashbox/multiprocessing_engine.py

```diff
@@ -208,6 +214,8 @@ def run():
         for p in _smash_.all_procs:
             p.join()
 
+        smashbox.utilities.finalize_test()
```

Comment:
Sorry @moscicki for creating a PR with your branch, just wanted to be able to comment it directly in line:

Are we able to set the exitcode inside the method?
Because in case our scrap log method found exceptions in the log, we want to return none-zero, so our jenkins enviroment can mark the test as failed.

Would be also nice if that can also be done on the setup_test, usecases:
* Check if sync client actually exists, is executable and has a given version: https://github.com/owncloud/smashbox/issues/99
* Check if provisioning API app is enabled: https://github.com/owncloud/smashbox/issues/126

If the checks fail executing the rest of the test also makes no sense at all.